src/protocol/ipa_prf/boolean_ops/addition_sequential.rs

```diff
@@ -183,7 +195,7 @@ mod test {
 
             let expected = (x + y) % (1_u128 << 64);
 
-            let result = world
+            let (_, result) = world
                 .semi_honest(records.into_iter(), |ctx, x_y| async move {
```

Comment:
It is a little odd that we need to put the two BA64 values into a Vec then produce an iterator.  Is there some reason this can't use `(x, y)`?